pmm-tests/pmm-2-0-bats-tests/generic-tests.bats

```diff
@@ -195,8 +185,6 @@ run pmm-admin summary --skip-server
 echo "$output"
     [ "$status" -eq 0 ]
     echo "${lines[-1]}" | grep ".zip created."
-    checkZipFileContents
```

Comment:
let's keep this test.

Reply:
@BupycHuk  this test exists, but we decided to remove files count check. or you think we should check it too?

Reply:
We don't have other test with the same amount of files

Reply:
we do actually, at line 174

Reply:
you are right, my bad